embedded_fab/main.dart

```diff
@@ -0,0 +1,416 @@
+                elevation: 0,
+                iconTheme: IconThemeData(color: grey800),
+              ),
+              body: Padding(
+                padding: const EdgeInsets.all(16.0),
+                child: Column(
+                  children: <Widget>[
+                    TextField(
+                      decoration: const InputDecoration(
+                        border: OutlineInputBorder(),
+                        filled: false,
+                        hintText: 'To',
+                      ),
+                    ),
+                    SizedBox(height: 8),
+                    TextField(
+                      decoration: const InputDecoration(
+                        border: OutlineInputBorder(),
+                        filled: false,
+                        hintText: 'Subject',
+                      ),
+                    ),
+                    SizedBox(height: 8),
+                    TextField(
+                      decoration: const InputDecoration(
+                        border: OutlineInputBorder(),
+                        filled: false,
+                        hintText: 'Compose email',
+                      ),
+                      maxLines: 10,
+                    ),
+                    SizedBox(height: 8),
+                  ],
+                ),
+              ),
+              extendBody: true,
+            );
+          },
+          fullscreenDialog: true,
+        ));
+  }
+
+
+  List<Widget> _drawerOptions() {
+    return [
+      Text('Options', style: Theme.of(context).textTheme.display1),
+      Divider(),
+      ListTile(title: Text('Shape')),
+      RadioListTile(
+        title: Text('Flat'),
+        value: ShapeType.Flat,
+        groupValue: _selectedShape,
+        onChanged: (value) {
+          _setSelectedTile(value);
+        },
+      ),
+      RadioListTile(
+        title: Text('Bump'),
+        value: ShapeType.Bump,
+        groupValue: _selectedShape,
+        onChanged: (value) {
+          _setSelectedTile(value);
+        },
+      ),
+      RadioListTile(
+        title: Text('Cut'),
+        value: ShapeType.Cut,
+        groupValue: _selectedShape,
+        onChanged: (value) {
+          _setSelectedTile(value);
+        },
+      ),
+      ListTile(title: Text('FAB Color')),
+      RadioListTile(
+        title: Text('White'),
+        value: Colors.white,
+        groupValue: _selectedColor,
+        onChanged: (value) {
+          _setSelectedColorTile(value);
+        },
+      ),
+      RadioListTile(
+        title: Text('Blue'),
+        value: blue500,
+        groupValue: _selectedColor,
+        onChanged: (value) {
+          _setSelectedColorTile(value);
+        },
+      ),
+      ListTile(title: Text('Layout')),
+      RadioListTile(
+          value: Layout.Experimental,
+          title: Text('Experimental'),
+          groupValue: _layout,
+          onChanged: (value) {
+            _setLayout(value);
+          }),
```

Comment:
To be consistent with examples above, you should add trailing comma here (and reformat)

```suggestion
          },),
```

Reply:
Same with the two items below here

Reply:
Done